File: MasterLogicaProgramacion/EjerciciosPython/Ej19.py
Object = lambda **kwargs: type("Object", (), kwargs)

peliculas = [
    Object(Titulo = "Percy Jackson", Director = "Pedro Dalton", Visto = True),
    Object(Titulo = "El Peluca", Director = "Peluca Peluquín", Visto = False),
    Object(Titulo = "El Hombre que Araña", Director = "Lionel Messi", Visto = True),
    Object(Titulo = "Jarry Póter", Director = "Cristina Fernández", Visto = False)
]

def peliculasVistas():
    vistas = [x.Titulo for x in peliculas if x.Visto]
    noVistas = [x.Titulo for x in peliculas if not x.Visto]

    return "Ya viste " + ", ".join(vistas) + "\nAún no viste " + ", ".join(noVistas)

print(peliculasVistas())

File: Cursos/MasterLogicaProgramacion/EjerciciosJavascript/Ej19.js
let peliculas = [
    {
        "Titulo": "Percy Jackson",
        "Director": "Pedro Dalton",
        "Visto": true
    },
    {
        "Titulo": "El Peluca",
        "Director": "Peluca Peluquín",
        "Visto": false
    },
    {
        "Titulo": "El Hombre que Araña",
        "Director": "Lionel Messi",
        "Visto": true
    },
    {
        "Titulo": "Jarry Póter",
        "Director": "Cristina Fernández",
        "Visto": false
    }
];

function peliculasVistas() {
    let vistas = peliculas.filter(x => x.Visto).map(x => x.Titulo);
    let noVistas = peliculas.filter(x => !x.Visto).map(x => x.Titulo);

    return "Ya viste " + vistas.join(", ") + "\nAún no viste " + noVistas.join(", ");
}

console.log(peliculasVistas());